spec/controllers/checkouts_controller_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+RSpec.describe CheckoutsController, type: :controller do
+  let(:user) { create(:user) }
+  let(:order_item) { create(:order_item) }
+  let(:order) { create(:order, order_items: [order_item]) }
+  let(:delivery) { create(:delivery) }
+
+  before do
+    allow(controller).to receive(:current_order).and_return(order)
+    allow(controller).to receive(:current_user).and_return(user)
+    sign_in(user)
+  end
+
+  describe '#show' do
+    context 'with :address step' do
+      it 'updates order status to in_progress' do
+        get :show, params: { id: :address }
```

Comment:
Maybe it would be better to cover your step by `step` key instead of `id` ?

Reply:
Wicked Gem has an `id` key by default.
https://github.com/zombocom/wicked#testing-with-rspec

Reply:
Ok, thanks for the clarification